spec/models/jitsi_call_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe JitsiCall, type: :model do
+  let(:jitsi_call) { FactoryBot.create :jitsi_call }
+
+  it 'is creatable using a factory' do
+    expect(jitsi_call).to be_valid
+  end
+
+  it 'is invalid without a room name' do
+    jitsi_call.room_name = ''
+    expect(jitsi_call).not_to be_valid
+  end
+
+  context 'with participants' do
+    let(:participants) { FactoryBot.create_list :user, 2 }
+
+    before do
+      participants.each { |p| jitsi_call.call_participants.create(user: p, state: 'accepted', role: 'participant') }
```

Comment:
Because the tests do not depend on a specific state or role, we could use the first one from their valid lists so we do not have to change the tests if the valid lists change.

Reply:
Fixed in 2b5f129. I also added a new do end thingy around it because the line got to long. Maybe you can do it better.